src/scripts/generic_coverage.py

```diff
@@ -329,8 +334,6 @@ def modify_docker_script(input_file, _output_file):
     cl_rel = Graph().parse("../ontology/test.ttl", format="ttl")
     cl = cl_base + cl_rel
 
-    cl.serialize("ontology.owl", format="turtle")
-
```

Comment:
Instead, you could save in the `../ontology/tmp/` folder

Reply:
It wasn't needed anyway. I saved it to check something for debug purposes.

Reply:
Better to add the test.ttl to the tmp folder, instead

Reply:
done